NEWS

```diff
@@ -1,3 +1,8 @@
+* crun-0.14.1
+
+- fix a regression in crun-0.14 where openat2(2) would fail when bind
+  mounting a symlink.
+
```

Comment:
Even though #430 is mostly not user-visible, maybe something like "various small fixes to allow running regression tests outside of source tree"?

Reply:
sure, added that.  Pushed a new version.